skeleton-typescript-webpack/tsconfig.json

```diff
@@ -9,6 +9,8 @@
   "filesGlob": [
     "./src/**/*.ts",
     "./node_modules/aurelia-*/dist/aurelia-*.d.ts",
+    "./custom_typings/**/*.d.ts",
```

Comment:
I've added custom typings bellow typings folder (typings/custom) to avoid bloating directory structure, don't You think this would be better?